Student/Canvas/Calendar/UI/View/CalendarView.swift

```diff
@@ -68,10 +68,16 @@ open class CalendarView: UIView, UICollectionViewDelegate, UICollectionViewDataS
 
     private static let dateFormatter: DateFormatter = {
         let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = DateFormatter.dateFormat(fromTemplate: "MMMM YYYY", options: 0, locale: Locale.current)
+        dateFormatter.dateFormat = DateFormatter.dateFormat(fromTemplate: "MMMM yyyy", options: 0, locale: Locale.current)
+        return dateFormatter
+    }()
+
+    private static let a11yMonthFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        // Parentheses prevent VO from pronouncing year 2020 as twentieth-twenty
+        dateFormatter.dateFormat = "MMMM (yyyy)"
```

Comment:
`DateFormatter.dateFormat(fromTemplate:)` strips non-template symbols like parentheses. I'm thinking the tradeoff here is that there could be locales where this would otherwise be formatted differently.